jni/tests/knn_extension/faiss/utils/BitSetTest.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "knn_extension/faiss/utils/BitSet.h"
+
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using ::testing::NiceMock;
+using ::testing::Return;
+using idx_t = faiss::idx_t;
+
+TEST(FixedBitSetTest, BasicAssertions) {
+    int ids1[4] = {3, 7, 11, 15};
```

Comment:
can you have values like 64, 128, 127, 1024, 34565 for your tests.

Reply:
ack